ToDo.DataAccess/ToDo.WebAPI/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ToDo.DataAccess;
+using ToDo.DataAccess.Models;
+using ToDo.DataAccess.Repositories.Interfaces;
+
+namespace ToDo.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoryController : ControllerBase
+    {
+        private readonly IGenericRepository<Category> categoryRepository;
```

Comment:
name private fields with the underscore at the beginning: `_categoryRepository`

Reply:
Changed.